client/styles/settings.less

```diff
@@ -300,6 +301,8 @@ section.mainSection {
 			font-size: 24px;
 			color: @trans-grey-56;
 			.transition01();
+			-webkit-transform-origin: 50% 50%;
+			transform-origin: 50% 50%;
```

Comment:
this is the default origin for transforms. Unfortunately it isnt working properly in opera, somehow the rotated icon is moved to the left and bottom, instead to be in the center. But it works good in chrome :)